### PR TITLE
Fire alarm changes done by Shabir

### PR DESCRIPTION
**Pull request by charithag**
Fire alarm changes done by Shabir for PC2 demo

### CODE REVIEW

File: modules/samples/firealarm/sketch/readTemperatureAndHumidity.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+"""
+
+import Adafruit_DHT             # Adafruit library required for temperature sensing
+
+
```
Comment (DilanUA) [inline modules/samples/firealarm/sketch/readTemperatureAndHumidity.py:24]: unnecessary new line.

File: modules/samples/firealarm/sketch/readTemperatureAndHumidity.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            print '{0:0.1f}C:{1:0.1f}%'.format(temperature, humidity)
+            # print 'RASPBERRY_STATS: Temp={0:0.1f}*C  Humidity={1:0.1f}%'.format(temperature, humidity)
+            # return (temperature + ":" + humidity)
+            return
+        except Exception, e:
+            print "RASPBERRY_STATS: Exception in TempReaderThread: Could not successfully read Temperature"
+            print ("RASPBERRY_STATS: " + str(e))
+            print '~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~'
+            pass
+
```
Comment (DilanUA) [inline modules/samples/firealarm/sketch/readTemperatureAndHumidity.py:46]: unnecessary new line.

File: modules/samples/firealarm/sketch/readTemperatureAndHumidity.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            time.sleep(3)
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+
```
Comment (DilanUA) [inline modules/samples/firealarm/sketch/readTemperatureAndHumidity.py:51]: unnecessary new lines.

File: modules/samples/firealarm/sketch/startservice.sh [hunk 4]
```diff
@@ -7,158 +7,19 @@ echo "|	       ----------------				"
...
-sudo nohup ./RaspberryStats.py -i $input </dev/null &
-
-if [ $? -ne 0 ]; then
-	echo "Could not start the service..."
-	exit;
+read -p "Whats the time-interval (in seconds) between successive Data-Pushes to the WSO2-IoT-Server (ex: '60' indicates 1 minute) > " interval
+
+if [ $interval -eq $interval 2>/dev/null ]
+then
+echo "Setting data-push interval to " $interval " seconds."
```
Comment (DilanUA) [inline modules/samples/firealarm/sketch/startservice.sh:16]: cannot we indent these lines to make them more readable?

File: modules/samples/firealarm/sketch/startservice.sh [hunk 5]
```diff
@@ -7,158 +7,19 @@ echo "|	       ----------------				"
...
+sed s/^push-interval=.*/push-interval=$interval/ deviceConfig.properties > myTmp
+mv -f myTmp deviceConfig.properties
+break;
 else
-	echo "Running the RaspberryAgent service...."
+echo "Input needs to be an integer indicating the number seconds between successive data-pushes."
```
Comment (DilanUA) [inline modules/samples/firealarm/sketch/startservice.sh:21]: cannot we indent these lines to make them more readable?

File: modules/samples/firealarm/sketch/switchBulb.py [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+"""
+
+import RPi.GPIO as GPIO
+import argparse
+
+BULB_PIN = 11               # The GPIO Pin# in RPi to which the LED is connected
+BULB_STATE = "OFF"
+
+
+
```
Comment (DilanUA) [inline modules/samples/firealarm/sketch/switchBulb.py:29]: unnecessary new lines.

File: modules/samples/firealarm/sketch/switchBulb.py [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+parser = argparse.ArgumentParser(description="Python service to switch LED on RPi")
+help_string = "switch-state of the BULB[LED] (default 'OFF')"
+parser.add_argument("-s", "--state", help=help_string)
+
+args = parser.parse_args()
+if args.state:
+    BULB_STATE = args.state
+### ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
```
Comment (DilanUA) [inline modules/samples/firealarm/sketch/switchBulb.py:43]: unnecessary new line.

File: modules/samples/firealarm/sketch/switchBulb.py [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+    except Exception as e:
+        print "IOT_UTILS: Exception at 'GPIO.setmode'"
+        print '~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~'
+        pass
+
+    GPIO.setup(BULB_PIN, GPIO.OUT)
+    GPIO.output(BULB_PIN, False)
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
```
Comment (DilanUA) [inline modules/samples/firealarm/sketch/switchBulb.py:60]: unnecessary new line.

File: modules/samples/firealarm/sketch/switchBulb.py [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+    elif state == "OFF":
+        GPIO.output(BULB_PIN, False)
+        print "BULB Switched OFF"
+
+    print '~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~'
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+
+
```
Comment (DilanUA) [inline modules/samples/firealarm/sketch/switchBulb.py:79]: unnecessary new lines.

**Comment by charithag** [comment]
Please kindly note that this commit is not belong to me. It is already stated in the commit massage.

File: modules/samples/firealarm/sketch/readTemperatureAndHumidity.py (hunk 1 above)
Comment (charithag) [inline modules/samples/firealarm/sketch/readTemperatureAndHumidity.py:24]: Can't fix as this code is obsolete and no longer exists in this repo.

File: modules/samples/firealarm/sketch/readTemperatureAndHumidity.py (hunk 2 above)
Comment (charithag) [inline modules/samples/firealarm/sketch/readTemperatureAndHumidity.py:46]: Can't fix as this code is obsolete and no longer exists in this repo.

File: modules/samples/firealarm/sketch/readTemperatureAndHumidity.py (hunk 3 above)
Comment (charithag) [inline modules/samples/firealarm/sketch/readTemperatureAndHumidity.py:51]: Can't fix as this code is obsolete and no longer exists in this repo.

File: modules/samples/firealarm/sketch/startservice.sh (hunk 4 above)
Comment (charithag) [inline modules/samples/firealarm/sketch/startservice.sh:16]: Can't fix as this code is obsolete and no longer exists in this repo.

File: modules/samples/firealarm/sketch/startservice.sh (hunk 5 above)
Comment (charithag) [inline modules/samples/firealarm/sketch/startservice.sh:21]: Can't fix as this code is obsolete and no longer exists in this repo.

File: modules/samples/firealarm/sketch/switchBulb.py (hunk 6 above)
Comment (charithag) [inline modules/samples/firealarm/sketch/switchBulb.py:29]: Can't fix as this code is obsolete and no longer exists in this repo.

File: modules/samples/firealarm/sketch/switchBulb.py (hunk 7 above)
Comment (charithag) [inline modules/samples/firealarm/sketch/switchBulb.py:43]: Can't fix as this code is obsolete and no longer exists in this repo.

File: modules/samples/firealarm/sketch/switchBulb.py (hunk 8 above)
Comment (charithag) [inline modules/samples/firealarm/sketch/switchBulb.py:60]: Can't fix as this code is obsolete and no longer exists in this repo.

File: modules/samples/firealarm/sketch/switchBulb.py (hunk 9 above)
Comment (charithag) [inline modules/samples/firealarm/sketch/switchBulb.py:79]: Can't fix as this code is obsolete and no longer exists in this repo.

**Pull request merged**